content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Notation v1.0.0-RC.4 is available!
+author:  "Notation Release Team"
+date: 2023-04-21
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of Notation CLI v1.0.0-rc.4, notation-go v1.0.0-rc.4, and notation-core-go v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds the following significant enhancements:
+
+- Support validating certificate revocation with [Online Certificate Status Protocol](https://datatracker.ietf.org/doc/html/rfc6960) (OCSP)
+- Introduce switch `NOTATION_EXPERIMENTAL=1` to enable experimental features
+- Introduce new CLI command `notation policy` to simplify trust policy configuration
+- Support OCI distribution referrers API
+- Introduce signing, listing and verification with [OCI image layout](https://github.com/opencontainers/image-spec/blob/v1.0/image-layout.md) as experimental feature
+
+### Support validating Certificate revocation with OCSP
+
+OCSP stands for Online Certificate Status Protocol. It is a protocol used to check the revocation status of digital certificates. When a certificate is revoked, it means that it is no longer valid and should not be trusted. To enforce validating certificate revocation, the verification level of trust policy configuration should be set to `strict` as following. Once this is done, the CLI command `notation verify` can be used to validate the signatures including checking certificate revocation status with OCSP. The verification process will fail if any certificate in the certificate chain has been revoked.
```

Comment:
Recommend using the following language and edits
Notation now supports Certificate Revocation using Online Certificate Status Protocol (OCSP) for distributing certificate revocation information. Notation clients uses OCSP to query if any certificate used for signing has been revoked. Signers can revoke a certificate to inform verifiers that any signature generated by the certificate (beyond the revocation date) should no longer be trusted. To enforce validating certificate revocation, the verification level of trust policy must be set to `strict`.  Depending on the trust policy verification level setting, `notation verify` command automatically validates the signatures and checks the revocation status. The verification process will fail if any certificate in the certificate chain has been revoked.